From 636f966ecc713c84ddd3b7ccfde2bfb2cc7492a0 Mon Sep 17 00:00:00 2001
From: Yann Collet <yann.collet.73@gmail.com>
Date: Sat, 4 Dec 2021 10:41:37 -0800
Subject: [PATCH 1/1] fix cmake --install

fix #646, reported by @kloczek .

Used this opportunity to add a `cmake --install` test in CI.
---
 .github/workflows/ci.yml        | 2 ++
 cmake_unofficial/CMakeLists.txt | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index 7c0b75f..746a1f9 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -253,6 +253,8 @@ jobs:
         cd build
         cmake ..
         CFLAGS=-Werror make
+        mkdir test_install_dir
+        DESTDIR=test_install_dir cmake --install .
 
 
   # Linux, { ARM, ARM64, PPC64LE, PPC64, S390X }
diff --git a/cmake_unofficial/CMakeLists.txt b/cmake_unofficial/CMakeLists.txt
index d5456b0..aca3d1f 100644
--- a/cmake_unofficial/CMakeLists.txt
+++ b/cmake_unofficial/CMakeLists.txt
@@ -122,7 +122,7 @@ if(NOT XXHASH_BUNDLED_MODE)
     install(TARGETS xxhsum
       EXPORT xxHashTargets
       RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
-    install(FILES "${XXHASH_DIR}/xxhsum.1"
+    install(FILES "${XXHSUM_DIR}/xxhsum.1"
       DESTINATION "${CMAKE_INSTALL_MANDIR}/man1")
   endif(XXHASH_BUILD_XXHSUM)
 
-- 
2.34.1

